lang/image.bzl

```diff
@@ -251,9 +255,10 @@ app_layer = rule(
 
         # Override the defaults.
         "data_path": attr.string(default = "."),
-        "workdir": attr.string(default = "/app"),
+        "workdir": attr.string(default = ""),
         "directory": attr.string(default = "/app"),
         "legacy_run_behavior": attr.bool(default = False),
+        "data": attr.label_list(cfg = "data", allow_files = True),
```

Comment:
iiuc, this should have as value host/target? https://docs.bazel.build/versions/master/skylark/lib/attr.html#label_list

Reply:
I have seen some doc referring that for data it should still be data, there is also a ticket for it: https://github.com/bazelbuild/bazel/issues/4316. But here: https://github.com/bazelbuild/examples/blob/5d1e5bd82e061e8e2e04bb54883fc4c79c935f76/rules/runfiles/execute.bzl#L32 there is nothing, so I will just remove it.